<launch>
<!-- Launch file for Livox AVIA LiDAR -->

	<arg name="rviz" default="true" />
	
	<rosparam command="load" file="$(find fast_lio)/config/avia.yaml" />

	<param name="feature_extract_enable" type="bool" value="0"/>
	<param name="point_filter_num" type="int" value="3"/>
	<param name="max_iteration" type="int" value="3" />
	<param name="filter_size_surf" type="double" value="0.2" />
	<param name="filter_size_map" type="double" value="0.2" />
	<param name="cube_side_length" type="double" value="1000" />
	<param name="runtime_pos_log_enable" type="bool" value="0" />
	<param name="map_path" type="string" value="/home/nv/drone_ws/fastlioplus_ws/src/FAST_LIO_LOCALIZATION_PLUS/PCD/scans.pcd" />
	<!-- <group ns="uav1">		 -->
	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_lio)/rviz_cfg/localization.rviz"/>
	</group>

<!-- <node pkg="fast_lio" type="fastlio_localization" name="fastlio_localization" output="screen" />  -->

<!-- Relocalization Handler Node -->

	<node pkg="fast_lio" type="relocalization_handler.py" name="relocalization_handler" output="screen"/>
	<node pkg="fast_lio" type="init_pose_pub.py" name="init_pose_pub" output="screen"/>
	<!-- </group> -->
</launch>
